includes/admin/js/perflab-install-activate-plugins.js

```diff
@@ -0,0 +1,53 @@
+/* eslint camelcase: "off", no-alert: "off" */
+/* global perflab_install_activate_plugins:false */
+
+( function ( document ) {
+	document.addEventListener( 'DOMContentLoaded', function () {
+		document.addEventListener( 'click', function ( event ) {
+			if (
+				event.target.classList.contains(
+					'perflab-install-active-plugin'
+				)
+			) {
+				const target = event.target;
+				target.classList.add( 'updating-message' );
+				target.innerHTML = wp.i18n.__( 'Activating…' );
```

Comment:
What about the text domain?

Reply:
Good catch, for the JS translations we can't easily use the core default text domain, so we should use `performance-lab` here instead of nothing

Reply:
Updated in https://github.com/WordPress/performance/pull/1091/commits/2630a2fea49e9c856bb59851ab3cf03e2529c179